ts/tsconfig.json

```diff
@@ -17,23 +17,13 @@
         "declaration": true,
         "composite": true,
         "target": "es6",
-        "module": "es6",
-        "lib": [
-            "es2017",
-            "es2018.intl",
-            "es2019.array",
-            "es2018.promise",
-            "es2020.promise",
-            "es2019.string",
-            "dom",
-            "dom.iterable"
-        ],
+        "module": "es2020",
+        "lib": ["es2020", "dom", "dom.iterable"],
```

Comment:
Is this safe? The current minimum Qt target is 5.14, which is Chromium 77 - which does not support things like the nullish coalescing operator. Neither does iOS 12 / early iOS 13.

Does this change enable something important that we were missing, or was it just about making things neater?

Reply:
In that case no, I'll revert it. I only needed to add support for [`.fromEntries`](https://caniuse.com/?search=fromEntries).
Is there some reference list of platforms Anki aims to support. I basically went through the lists on `icanuse` and looked out for:
- support for Chromium 83
- support for iOS 14/15
in which case we could go with es2020.

Reply:
(it might be worth setting up eslint to warn us when we use something outside those versions - feel free to look into that if you're interested :-)

Reply:
#982 tracks the 5.15 issues